analysis/log_file_analyzer.py

```diff
@@ -53,6 +52,7 @@ def normalize_logs(logs):
     logs['LON'] = logs['LON'].apply(lambda x: x * 0.0000001)
     logs['GSPEED'] = logs['GSPEED'].apply(lambda x: x * 0.01)
     logs['CRS'] = logs['CRS'].apply(lambda x: x * 0.00001)
+    return logs
```

Comment:
you do not need to return the 'logs' object because it's the same object as it was in the beginning of the function.
I mean that changes are inplace.

Reply:
I did it when testing with js parsing. I will remove it